### DISCUSSION TITLE
Admin dashboard for plugnmeet

### DISCUSSION THREAD

**Discussion by shubhendumadhukar** [mynaparrot/plugNmeet-server#218] [Ideas]
Hello,

I have a proposal for a new plugnmeet app. Is there is any current plan to build an administrative panel, or would you be interested in it?

It would be a space where we could get the information about the meetings created in plugnmeet.

Information like meetings created, their durations, number of participants allowed, if the meeting started and if it did, how long did it run,  how many participants joined. Basically kind of a separate analytics server that gets data from the plugnmeet server (optionally).

I understand it would be big effort but it could be an useful app in plugnmeet ecosystem. I would be happy to contribute if this is something that sounds interesting to you.

**Comment by jibon57** [comment]
Thank you. We don't have any plans for building separate interface but have plan for analytic reports in near future. Later implement API for fatching info.
